test/integration/suites/upgrade/versions.txt

```diff
@@ -1,3 +1,4 @@
 1.1.0
 1.1.1
 1.1.2
+1.1.3
```

Comment:
We were missing 1.1.3 already :) We should add 1.2.0 now.

Reply:
Aaah! Thanks @amartinezfayo :) it is now fixed